### PR TITLE
Migrate fileupload

### PR DESCRIPTION
**Pull request by moka1812**

### CODE REVIEW

**Comment by Git-on-my-level** [comment]
You should make sure CI passes before adding reviewers

File: packages/sdk-api/test/fileupload.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <p id="p"></p>
+
+</body>
+<script>
+    const inputElement = document.getElementById("inputform");
+    inputElement.addEventListener("change", handleFiles, false);
+    async function handleFiles() {
+        console.log("uploading")
+        const url = 'http://localhost:5000/fileupload/document'
+        const token =
```
Comment (Git-on-my-level) [inline packages/sdk-api/test/fileupload.html]: hardcoded?

**Comment by Git-on-my-level** [comment]
Try not to leave PRs open for too long, or if they are pending something you should note it here

File: packages/sdk-api/src/services/index.ts [hunk 2]
```diff
@@ -122,6 +128,37 @@ export class Api {
...
     return this.getJson(response)
   }
+  /***
+   * form sends data to the API
+   * this will be an HTTPS POST call
+   * with Authorization header (if authenticated)
+   * with a multipart/formdata body - Using for fileupload
+   * returns json
+   */
+  public async form(url: string, file: any): Promise<JsonData | undefined> {
```
Comment (adamgering) [inline packages/sdk-api/src/services/index.ts]: where is this being called?

File: packages/sdk-react/src/forms/kyc.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { useFormStateAction } from '../hooks/form'
+import { ActionUserAddress } from '@stably/sdk-api'
+
+export const KycForm: React.FC = () => {
+  //const action = new ActionUserName()
+  // setValues, setErrors,
+  const { enabled, handleChange, handleSubmit } = useFormStateAction(new ActionUserAddress())
```
Comment (adamgering) [inline packages/sdk-react/src/forms/kyc.tsx]: Why useFormStateAction?

**Pull request closed without merging**